types/run.sh

```diff
@@ -36,7 +36,7 @@ pyann-patched() {
 
 iter-demo-asdl() {
   asdl/run.sh gen-typed-demo-asdl
-  typecheck --strict \
+  typecheck $MYPY_FLAGS \
```

Comment:
I think the intention was that ASDL code gen is strict, but the rest of Oil is not strict.

That will make it easier to turn on `--strict-optional later`.  That is ASDL won't regress while we work on the OIl codebase in general.